app/parse.py

```diff
@@ -8,8 +15,60 @@ class Quote:
     tags: list[str]
 
 
+QUOTE_FIELDS = ["text", "author", "tags"]
+
+HOME_PAGE = "https://quotes.toscrape.com/"
+
+
+def parse_single_quote(quote_element: Tag) -> [Quote]:
```

Comment:
The return type annotation for `parse_single_quote` should be `Quote` instead of `[Quote]` since it returns a single `Quote` object, not a list.